jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -1,3 +1,8 @@
+def getBuild(job_url) {
```

Comment:
Can we add this function after pipeline code?

Reply:
Added and tested with - 
http://eos-jenkins.mero.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nikhildev/job/Nightly-K8s-Build-and-Deploy-test/75/console